### DISCUSSION TITLE
Is it possible to use Supabase authentication only, without using Supabase’s database and storage?

### DISCUSSION THREAD

**Discussion by Sidney11a** [supabase/supabase#6426] [Questions] [answered]
We are really interested in integrating Supabase. We’ve been checking out your products for the last couple weeks and have a few questions:
 
Is it possible to use Supabase authentication only, without using Supabase’s database and storage? What are the implications of this? We currently use AWS for storage and database but we are looking for an authentication product and are considering Supabase for this.
 
I’ve been trying to find a video/demo showing Supabase’s user management dashboard but I can’t seem to find one. Does the user management dashboard show you details such as which users logged in to the app, when they logged in to the app, how they logged in, did a user change their password, failed log in attempts etc?

**Comment by carloslfu** [comment, marked as answer]
@Sidney11a. I just started doing this and it seems to work fine, Supabase Auth can be used in isolation. Indeed, it works in isolation (it saves your user data in `auth.users` which is another DB schema, not `public`). I am not sure about the other questions, but on my side, depending on the use case I would collect those events using a product analytics tool such as Mixpanel or a logging tool such as Datadog.

**Comment by J0** [comment]
Hey Supabase engineer here,

I would like to re-affirm that it is possible to use Supabase Auth as a standalone service without making use of the Database and Storage services.  Do encourage you to check out our other services as well though. Auth can be used standalone but it is even better when used in conjunction is other services

**Reply by OsamaQureshi147** [reply]
Hey there,
I am using Supabase auth service but using my own postgresql database for the rest of the app. 
How can I sync up my local postgress DB with supabase `auth.users` table. I have tried implementing foreign tables in our local postgress but that is not helping in creating relationships.
I have also tried adding my own postgress pool inside supabase that triggers an insert operation whenever a new user sign's up. But here I am getting TCP/IP connection error (I have verified all security options of my own postgress).
I am badly stuck here and after so much effort l am looking to rollback to some other solution. 
Please let me know if you have any idea.

**Comment by semanser** [comment]
@Sidney11a I just posted an extensive guide on how to do it. It uses Go on the backend but you should be able to adjust it to any language/framework if needed. https://depshub.com/blog/using-supabase-auth-as-a-service-with-a-custom-backend/ The source code is available [here](https://github.com/DepshubHQ/supabase-jwt-auth-demo).
